src/olympia/lib/settings_base.py

```diff
@@ -1873,6 +1873,8 @@ def get_raven_release():
 CUSTOMS_API_KEY = env('CUSTOMS_API_KEY', default=None)
 WAT_API_URL = env('WAT_API_URL', default=None)
 WAT_API_KEY = env('WAT_API_KEY', default=None)
+ML_API_URL = env('ML_API_URL', default=None)
```

Comment:
A longer prefix would be ideal.  A two-letter abbreviation could mean many things.

Reply:
I agree it's not the best name ever.. I left a note about that in this PR's description. I think I'll change that once we have a codename.